.github/CI.yml

```diff
@@ -0,0 +1,120 @@
+# This file is autogenerated by maturin v1.2.3
+# To update, run
+#
+#    maturin generate-ci github
+#
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64, x86, aarch64, armv7, s390x, ppc64le]
```

Comment:
i believe you can skip s390x in the pipeline, it was discontinued in 2004 😄 same goes to ppc64le, I suggest leaving only:

x86_64, i386/i686, apple-aarch64-darwin, armv7, aarch64/arm64v8

Reply:
remark: I didn't notice that it was generated :D later on, stick to same targets as our CI

Reply:
sgtm @hubcio 😄